dts/arm/st/g4/stm32g431X8.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 Guðni Már Gilbert
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/g4/stm32g431.dtsi>
```

Comment:
nit: this change should likely belong to 71170107f470681ce23df61e2a0a8843f0a1c8a2

Reply:
Good catch! 💪you’re right.